src/traits.rs

```diff
@@ -31,6 +32,18 @@ pub trait AudioTagEdit: AudioTagConfig {
         self.set_artist(artist);
     }
 
+    fn original_date_released(&self) -> Option<Timestamp>;
+    fn set_original_date_released(&mut self, date: Timestamp);
+    fn remove_original_date_released(&mut self);
+
+    fn date_released(&self) -> Option<Timestamp>;
+    fn set_date_released(&mut self, date: Timestamp);
+    fn remove_date_released(&mut self);
```

Comment:
```suggestion
```

With the removal of the fields from `AnyTag`, these methods should be taken out as well.

Reply:
Ha, you beat me to it. Just removed them. Thanks!